hera_opm/mf_tools.py

```diff
@@ -166,15 +210,17 @@ def make_time_neighbor_outfile_name(obsid, action, obsids, pol=None, n_neighbors
     obsid : str
         The obsid of the current file.
     action : str
-        The action corresponding to the time prereqs.
+        The action corresponding to the prereqs.
     obsids : list of str
         A list of all obsids for the given day; uses this list (sorted) to
         define neighbors
-    pol : str, optional
-        If present, polarization string to specify for output file.
-    n_neighbors : str
+    n_time_neighbors : str
```

Comment:
Thanks for going through and making sure we're self-consistent in our variable names.